mapper-pojo/src/main/java/org/hibernate/search/v6poc/entity/pojo/bridge/spi/FunctionBridge.java

```diff
@@ -23,12 +23,12 @@ default void initialize(BuildContext buildContext) {
 		return null; // Auto-detect the return type and use default encoding
 	}
 
-	R apply(T propertyValue);
+	R toFieldValue(T propertyValue);
```

Comment:
So, I'm definitely +1 on this change. I'm just not totally sold on the `Field` part, considering we sometimes call the entity property a field too. Maybe `toIndexedValue()`/`fromIndexedValue()`?

Reply:
Right, will do.